### PR TITLE
chore: Update patches description

### PR DESCRIPTION
**Pull request by PalmDevs**
This commit updates patches description to make it more clear to new users

### CODE REVIEW

File: src/main/kotlin/app/revanced/patches/youtube/misc/microg/patch/bytecode/MicroGBytecodePatch.kt [hunk 1]
```diff
@@ -37,7 +37,7 @@ import org.jf.dexlib2.immutable.reference.ImmutableStringReference
     ]
 )
 @Name("microg-support")
-@Description("Allows YouTube ReVanced to run without root and under a different package name.")
+@Description("Allows YouTube ReVanced to run without root and under a different package name. But, you need Vanced MicroG in order to sign in.")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/misc/microg/patch/bytecode/MicroGBytecodePatch.kt:40]:
```suggestion
@Description("Allows YouTube ReVanced to run without root and under a different package name with Vanced MicroG")
```

File: src/main/kotlin/app/revanced/patches/youtube/layout/oldqualitylayout/patch/OldQualityLayoutPatch.kt [hunk 2]
```diff
@@ -22,7 +22,7 @@ import org.jf.dexlib2.builder.instruction.BuilderInstruction21t
 @Patch
 @Dependencies([IntegrationsPatch::class])
 @Name("old-quality-layout")
-@Description("Enables the original quality flyout menu.")
+@Description("Enables the original quality flyout menu. Which lets you select video quality immediately.")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/layout/oldqualitylayout/patch/OldQualityLayoutPatch.kt:25]:
```suggestion
@Description("Enables the original quality flyout menu.")
```

File: src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/patch/EnableSeekbarTappingPatch.kt [hunk 3]
```diff
@@ -24,7 +24,7 @@ import org.jf.dexlib2.iface.instruction.formats.Instruction35c
 @Patch
 @Dependencies([IntegrationsPatch::class])
 @Name("seekbar-tapping")
-@Description("Enables tapping on the seekbar of the YouTube player.")
+@Description("Enables tap-to-seek on the seekbar bar of the video player.")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/youtube/interaction/seekbar/patch/EnableSeekbarTappingPatch.kt:27]:
```suggestion
@Description("Enables tap-to-seek on the seekbar of the video player.")
```

File: src/main/kotlin/app/revanced/patches/music/audio/exclusiveaudio/patch/ExclusiveAudioPatch.kt [hunk 4]
```diff
@@ -17,7 +17,7 @@ import app.revanced.patches.music.audio.exclusiveaudio.fingerprints.ExclusiveAud
 
 @Patch
 @Name("exclusive-audio-playback")
-@Description("Adds the option to play music without video.")
+@Description("Adds an option to play music without video.")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/music/audio/exclusiveaudio/patch/ExclusiveAudioPatch.kt:20]:
```suggestion
@Description("Enables the option to play music without video.")
```

File: src/main/kotlin/app/revanced/patches/music/audio/codecs/patch/CodecsUnlockPatch.kt [hunk 5]
```diff
@@ -16,7 +16,7 @@ import app.revanced.patches.music.audio.codecs.fingerprints.CodecsLockFingerprin
 
 @Patch
 @Name("codecs-unlock")
-@Description("Enables more audio codecs. Usually results in better audio quality but may depend on song and device.")
+@Description("Adds more audio codec options. Changing audio codecs usually results in better audio quality but may depend on song and device.")
```
Comment (oSumAtrIX) [inline src/main/kotlin/app/revanced/patches/music/audio/codecs/patch/CodecsUnlockPatch.kt:19]:
```suggestion
@Description("Adds more audio codec options. The new audio codecs usually result in better audio quality.")
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

**Comment by PalmDevs** [comment]
@oSumAtrIX Please squash this PR

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**